java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -307,7 +307,15 @@ public final int getNumChildren() {
    * Returns the amount of device memory used.
    */
   public long getDeviceMemorySize() {
-    return getDeviceMemorySize(getNativeView());
+    return getDeviceMemorySize(getNativeView(), false);
+  }
+
+  /**
+   * Returns the amount of memory used by this, but padded for 64-bit alignment. This makes it
+   * so it could be used as the amount of memory needed to copy the data to the host.
+   */
+  public long getDeviceMemorySizeAligned() {
```

Comment:
nit: can we incorporate 64 in the method name?

```suggestion
  public long getDeviceMemorySizeAligned64() {
```
or 

```suggestion
  public long getDeviceMemorySizePadded64() {
```